src/main/java/com/plaid/client/request/InstitutionsGetRequest.java

```diff
@@ -53,9 +53,35 @@ public InstitutionsGetRequest withOAuth(boolean oauth) {
     return this;
   }
 
+  public InstitutionsGetRequest withIncludePaymentInitiationMetadata(boolean includePaymentInitiationMetadata) {
+    if (this.options == null) {
+      this.options = new Options();
+    }
+    this.options.includePaymentInitiationMetadata = includePaymentInitiationMetadata;
+    return this;
+  }
+
+  public InstitutionsGetRequest withPaymentInitiation(PaymentInitiation paymentInitiation) {
```

Comment:
This isn't supported for `InstitutionsGet`, we decided to not include this filter since it messes with the pagination for this route

Reply:
Good catch https://github.com/plaid/plaid-java/pull/281/commits/4e85203ad1913a41be4e6fef97b9706c6a06f5ce